homeassistant/components/fritz/common.py

```diff
@@ -133,7 +135,7 @@ def signal_device_update(self) -> str:
         """Event specific per FRITZ!Box entry to signal updates in devices."""
         return f"{DOMAIN}-device-update-{self._unique_id}"
 
-    def _update_info(self):
+    def _update_info(self) -> Any:
```

Comment:
```suggestion
    def _update_info(self) -> list[HostInfo]:
```
where `HostInfo` is:
```python
class HostInfo(TypedDict):
    mac: str
    name: str
    ip: str
    status: bool
```

Reply:
Done